packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -1555,6 +1562,7 @@ export type ChatInitializedListener = (event: {
 export interface ChatMessage extends MessageCommon {
     // (undocumented)
     attached?: MessageAttachedStatus;
+    attachments?: AttachmentMetadata[];
```

Comment:
I'm unclear as to why we're stabilizing this specific property - it doesn't seem to add any stabilized feature?